### ISSUE TITLE
Some cursor themes are not scaled correctly

### ISSUE THREAD

**Issue by zerodegress** [hyprwm/hyprcursor#68]
Some cursor themes  converted from xcursor like [https://www.gnome-look.org/p/2113061](https://www.gnome-look.org/p/2113061) are not scaled correctly, which makes the cursor appear abnormally small on high-resolution screens

**Comment by vaxerski** [comment]
that'd be an issue with xcur2png tho no?

**Comment by zerodegress** [comment]
> that'd be an issue with xcur2png tho no?
I don't think so, because it's also a problem when using the XCursor mouse theme under GTK applications

**Comment by vaxerski** [comment]
maybe its a problem with the theme then? if only select themes are affected

**Comment by zerodegress** [comment]
> maybe its a problem with the theme then? if only select themes are affected

In fact all themes in this category suffer from these problems. It seems that highly colorful, pixelated and dynamic mouse themes will fail to scale properly, but vector graphic mice in general are not affected.

**Comment by PacificViking** [comment]
I am having the same issue with the animated cursor https://www.gnome-look.org/p/2163432. The converted cursor only loads at specific values of HYPRCURSOR_SIZE.

(I suspect this is because there is no upscaling done on the png when it has low resolution, because reported issues seem to be with low base resolution pixel-style cursors)
